### PR TITLE
fix typo in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by mladedav**
## Motivation

There was an unclosed parenthesis in the CONTRIBUTING.md document.

## Solution

I removed said parenthesis.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -267,7 +267,7 @@ If a Pull Request appears to be abandoned or stalled, it is polite to first
 check with the contributor to see if they intend to continue the work before
 checking if they would mind if you took it over (especially if it just has nits
 left). When doing so, it is courteous to give the original contributor credit
-for the work they started (either by preserving their name and email address in
+for the work they started either by preserving their name and email address in
```
Comment (jplatte) [inline CONTRIBUTING.md]:
```suggestion
for the work they started, either by preserving their name and email address in
```

**Review by jplatte — APPROVED** [review]
Thanks for spotting!

**Comment by jplatte** [comment]
I'll take care of the CI failure in a separate PR.

**Comment by mladedav** [comment]
Thanks, I've noticed it started happening lately but I didn't track the duplicates in examples.

**Comment by jplatte** [comment]
Can you rebase to fix the CI issue?

**Comment by mladedav** [comment]
@jplatte Done

**Pull request merged**